cmd/cosign/cli/options/attest_blob.go

```diff
@@ -107,5 +108,5 @@ func (o *AttestBlobOptions) AddFlags(cmd *cobra.Command) {
 
 	cmd.Flags().StringVar(&o.RFC3161TimestampPath, "rfc3161-timestamp-bundle", "",
 		"path to an RFC 3161 timestamp bundle FILE")
-	_ = cmd.Flags().SetAnnotation("rfc3161-timestamp-bundle", cobra.BashCompFilenameExt, []string{})
+	// _ = cmd.MarkFlagFilename("rfc3161-timestamp-bundle") // no typical extensions
```

Comment:
To confirm, is there any purpose to including this? As in, will shell completion still work with `rfc3161-...` for example?

Reply:
This is discussed in detail in the aa5537c commit message.

Reply:
But TL;DR: yes :)